ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -83,25 +83,24 @@ export default function OnboardingMetametrics() {
         className="onboarding-metametrics"
         data-testid="onboarding-legacy-metametrics"
       >
-        <Typography
-          variant={TypographyVariant.H2}
+        <Text
+          variant={TextVariant.headingLg}
           align={TEXT_ALIGN.CENTER}
```

Comment:
Devin needs to be able to spot differences in component api and update accordingly. In the `Text` component `align` has been changed to `textAlign`. It should also use the enum TextAlign instead of the const
```suggestion
          textAlign={TextAlign.Center}
```